scenarioo-client/test/protractorE2E/specs/manage_branch_aliases.ts

```diff
@@ -1,4 +1,5 @@
 'use strict';
+import {scenario, step, useCase} from "scenarioo-js";
 
 var scenarioo = require('scenarioo-js');
```

Comment:
we should as well migrate this requires to imports, shouldnt we?
why is this `scenarioo` used anyway here ??

Reply:
dito for all other test files - maybe open an issue for future - if not resolvable immediately. Thanks.

Reply:
dead import